espnet2/enh/loss/criterions/time_domain.py

```diff
@@ -1,5 +1,6 @@
 from abc import ABC
 
+import fast_bss_eval
```

Comment:
To make this work, we need to ensure the package is installed.
Could you add it to [setup.py](https://github.com/espnet/espnet/blob/master/setup.py#L36)?